Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1853,6 +1854,7 @@ - (void)updateAppThemeIfNecessary {
     if (self.theme != theme) {
         [self applyTheme:theme];
         [self.settingsViewController loadSections];
+        [self setWKAppEnvironmentThemeWithTheme: theme];
```

Comment:
Just a note for forthcoming PRs that we'd eventually need to also pass along the trait collection changes so the font scaling respects our design intents.

Reply:
Thanks - this is updated!

Reply:
good stuff